pom.xml

```diff
@@ -63,6 +63,9 @@
       <tag>HEAD</tag>
     </scm>
     <properties>
+        <jahia-deploy-on-site>all</jahia-deploy-on-site>
```

Comment:
what is this for ?

Reply:
I saw all the recent javascript module doing this, and since the CKEditor js is already deployed to every jahia ui on any site, like this module, I just wanted to be consistent.
Like: 
- https://github.com/Jahia/jcontent/blob/master/pom.xml#L68
- https://github.com/Jahia/jahia-repository-explorer/blob/master/pom.xml#L68
- https://github.com/Jahia/jahia-page-composer/blob/master/pom.xml#L68
- https://github.com/Jahia/jahia-dashboard/blob/master/pom.xml#L69
- https://github.com/Jahia/jahia-category-manager/blob/master/pom.xml#L68
- https://github.com/Jahia/jahia-administration/blob/master/pom.xml#L68

If you think it's doest make sense, I will remove it.

Reply:
Actually I've seen it recently on all the modules you mention, as it has a strange and unexpected side-effect on the remote publish . I don't think it's needed in any of these modules .. so to be consistent I'd rather remove it everywhere :-)  ( if it's really not needed .. I did not have the time to check )

Reply:
After check I think it's useless, so I removed it here.